server/subscribe.go

```diff
@@ -360,22 +367,167 @@ func (p *Plugin) listChannelSubscriptions() (string, error) {
 		return "", err
 	}
 
+	sortedSubs, err := p.getSubscriptionsSorted()
```

Comment:
sorry to be a nitpicker, but for consistency (and readability) would it be better `getSortedSubscriptions`?

Reply:
I thought about this for a bit and I agree, this should be changed, mainly for readability.  It's clearer with <adjective><nown>